react-native-in-app-payments-quickstart/app/service/CreateCustomerCard.tsx

```diff
@@ -17,7 +17,7 @@
 import { CREATE_CUSTOMER_CARD_SERVER_URL } from '../Constants';
 import CreateCustomerCardError from '../CreateCustomerCardError';
 
-export default async function createCustomerCard(customer_id, nonce) {
+export default async function createCustomerCard(customer_id: any, nonce: any) {
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.